.changeset/giant-olives-protect.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/backend-app-api': patch
+---
+
+fixed bug the treat as same as no cookie if existing cookie vas invalid.
```

Comment:
```suggestion
Fixed a bug where expired cookies would not be refreshed.
```

Reply:
I'm sorry about my poor English.
I'll fix it.